docs/_docs/sql-reference/operational-commands.adoc

```diff
@@ -113,3 +113,120 @@ After executing the above command and *closing the JDBC connection*, all data wi
 image::images/set-streaming.png[]
 
 
+== KILL QUERY
+
+The `KILL QUERY` command allows you to cancel a running query.
+
+[source,sql]
+----
+KILL QUERY {ASYNC} '<query_id>'
+----
+
+=== Description
+`ASYNC` is an optional parameter which returns control immediately without waiting for the cancellation to finish, and `query_id` can be retrieved via the `SQL_QUERIES` view.
+
+When a query cancelled via the `KILL` command, all parts of the query running on all other nodes are cancelled.
+
+This command available via JMX and CMD, also:
+
+- `QueryMXBean#cancelSQL`
+- `./control.sh --kill SQL query_id`
+
+== KILL TRANSACTION
+
+The `KILL TRANSACTION` command allows you to cancel a running transaction.
+
+[source,sql]
+----
+KILL TRANSACTION 'xid'
+----
+
+=== Description
+`xid` of the transaction can be retrived via the `TRANSACTIONS` view.
+
+This command available via JMX and CMD, also:
+
+- `TransactionMXBean#cancel`
+- `./control.sh --kill TRANSACTION xid`
```

Comment:
I would add an explanation on how to find the "xid" that needs to be passed into this command.

Reply:
I explained it few lines above:

`xid` of the transaction can be retrived via the `TRANSACTIONS` view.